configs/_base_/datasets/synapse.py

```diff
@@ -0,0 +1,41 @@
+dataset_type = 'SynapseDataset'
```

Comment:
The setting of this synapse dataset config may be aligned with TransUNet default setting: https://github.com/Beckschen/TransUNet/blob/main/train.py#L19-L33.